app/lib/ability.rb

```diff
@@ -290,6 +257,42 @@ def facility_senior_staff_abilities(user, resource, controller)
 
 
   def facility_staff_abilities(user, resource, controller)
+    can :available_for_cross_core_ordering, Product if controller.is_a?(FacilityOrdersController) || controller.is_a?(ProductsController)
```

Comment:
curious if this is necessary since we are skip the authorize resource call?

Reply:
Yes, it's still necessary. If I remove this, specs fail as no user is allowed to hit that endpoint